monolith/views/test/test_stories.py

```diff
@@ -66,15 +66,22 @@ def test_delete_story(self):
         reply = self.client.get('/story/1/delete')
         self.assert_template_used("message.html")
         self.assert_context("message", "Story sucessfully deleted")
+        with self.context:
+            s = Story.query.filter_by(id=1).first()
+            self.assertIsNone(s)
         
     def test_random_story(self):
     
         # success: render the random story
-        #TODO
+        self._signup("fantastic@example.com", "betterNerfIrelia", "404", "404", "01/01/1964", True)
+        self._login("fantastic@example.com", "betterNerfIrelia")
+        reply = self.client.post('/stories/new_story', data={ "theme" : "Mountain", "dice_number" : "3"})
+        #TODO publish story first!
+        self._logout()
         #self._login("example@example.com", "admin")
         #reply = self.client.get('/random_story')
         #self.assert_template_used("story.html")
-        #self.assert_context("rolls_outcome", "[]")
+        #self.assert_context("message", "")
```

Comment:
just remove them if we're not using/doing that

Reply:
Done